mambaSharedFramework/Rapid Parser/RapidParser.h

```diff
@@ -20,10 +20,10 @@
 @import Foundation;
 #include "RapidParserError.h"
 
-@protocol HLSRapidParserCallback;
+@protocol RapidParserCallback;
 
-@interface HLSRapidParser : NSObject
+@interface RapidParser : NSObject
 
-- (void)parseHLSData:(NSData * _Nonnull)data callback:(id<HLSRapidParserCallback> _Nonnull)callback;
+- (void)parseHLSData:(NSData * _Nonnull)data callback:(id<RapidParserCallback> _Nonnull)callback;
```

Comment:
Entire file, `PlaylistRapidParser...`?

Reply:
I think the context is enough for the RapidParser (It's private to mamba.)